### PR TITLE
New game UI blacksmith sidebar

### PR DESCRIPTION
**Pull request by Spintacle123**
PR Includes:
 - Blacksmith new sidebar

### CODE REVIEW

File: frontend/src/components/smart/CharacterDisplay.vue [hunk 1]
```diff
@@ -91,7 +119,41 @@ export default Vue.extend({
   data() {
     return {
       traits: CharacterTrait,
-      isPlaza : false
+      isPlaza : false,
+      sideBarBlacksmith: [
+        {
+          id: 0,
+          title: 'Weapon',
```
Comment (kevinunger) [inline frontend/src/components/smart/CharacterDisplay.vue:126]: add i18n translation

Comment (Sellie96) [reply frontend/src/components/smart/CharacterDisplay.vue:126]: This still needs doing

File: frontend/src/App.vue [hunk 2]
```diff
@@ -229,7 +231,11 @@ export default {
         if(this.toggleSideBar){
           toDisplay = 'can-show-app';
         }else{
-          toDisplay = 'col-xl-9 col-lg-8 col-md-8 col-sm-10 cols-11 set-normal';
+          if(this.currentPath === '/blacksmith'){
+            toDisplay = 'col-xl-10 col-lg-9 col-md-9 col-sm-10 cols-11 set-normal';
+          }else{
+            toDisplay = 'col-xl-9 col-lg-8 col-md-8 col-sm-10 cols-11 set-normal';
```
Comment (Sellie96) [inline frontend/src/App.vue:237]: We do not want two different size sidebars. Choose one size for all. I think you want the top one as default

File: frontend/src/components/CharacterBar.vue [hunk 3]
```diff
@@ -24,20 +24,36 @@ import CharacterDisplay from './smart/CharacterDisplay.vue';
...
+    renderPageDisplay(){
+      let toDisplay;
+
+      if(this.isToggled){
+        toDisplay = 'sideBorder';
+      }else{
+        if(this.currentPath === '/blacksmith'){
+          toDisplay = 'col-xl-2 col-lg-3 col-md-3 col-sm-2 cols-1';
+        }else{
+          toDisplay = 'col-xl-3 col-lg-4 col-md-4 col-sm-2 cols-1';
```
Comment (Sellie96) [inline frontend/src/components/CharacterBar.vue:52]: Same as previous comment

File: frontend/src/components/NavBar.vue [hunk 4]
```diff
@@ -76,12 +77,29 @@ export default Vue.extend({
         this.getIsCharacterViewExpanded ? 'true' : 'false'
       );
     },
+    renderPageDisplay(){
+      let toDisplay;
+
+      if(this.currentRoute === '/blacksmith'){
+        toDisplay = 'col-xl-2 col-lg-3 col-md-3 col-sm-2 cols-1';
+      }else{
+        toDisplay = 'col-xl-3 col-lg-4 col-md-4 col-sm-2 cols-1';
```
Comment (Sellie96) [inline frontend/src/components/NavBar.vue:86]: And again

File: frontend/src/components/smart/CharacterDisplay.vue [hunk 5]
```diff
@@ -4,41 +4,69 @@
...
+                      v-tooltip.bottom="toolTipHtml(timeUntilCharacterHasMaxStamina(c.id))">
+                    </div>
+                    <div class="char-level">
+                        {{$t('PlayToEarn.level')}} {{ c.level + 1}} <span> (STA {{ getCharacterStamina(c.id) }} / 200)</span>
+                    </div>
+                  </div>
+                </div>
+          </b-col>
+          <b-col class="character-list centered-list" v-else>
+                <div v-for="s in sideBarBlacksmith"
```
Comment (Sellie96) [inline frontend/src/components/smart/CharacterDisplay.vue:45]: s is too verbose can you make it clear what you are looping on at a glance it is not clear what it is.

**Review by Sellie96 — APPROVED** [review]

**Pull request merged**